src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,16 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+// @codingStandardsIgnoreFile
+namespace Cake\Network\Email;
+
+class_alias(\Cake\Mailer\AbstractTransport::class, AbstractTransport::class);
+
+if (class_exists(AbstractTransport::class)) {
+    return;
+}
```

Comment:
@markstory phpstorm does not support the class_alias, and this class stub is just there to provide a fix. The if condition is there to stop PHP from parsing the duplicate class. If I use `if (true)` to always return phpstorm sees that and does not read the class definition below it.

I'm more challenged and need auto-complete in my IDE. 😄